src/query/v2/trigger_context.hpp

```diff
@@ -223,8 +227,14 @@ class TriggerContextCollector {
   struct HashPairWithAccessor {
     template <detail::ObjectAccessor TAccessor, typename T2>
     size_t operator()(const std::pair<TAccessor, T2> &pair) const {
-      using GidType = decltype(std::declval<TAccessor>().Gid());
-      return utils::HashCombine<GidType, T2>{}(pair.first.Gid(), pair.second);
+      // TODO(jbajic) Fix Remove Gid
+      if constexpr (std::is_same_v<TAccessor, VertexAccessor>) {
+        static uint64_t i{0};
+        return utils::HashCombine<uint64_t, T2>{}(i++, pair.second);
```

Comment:
Why not just `kFakeVertexGid` instead of `i++`?

Reply:
It wasn't much of a difference just wanted t omake code compilable, but will change it